src/redux/configureStore.js

```diff
@@ -1,15 +1,31 @@
 import { createStore, combineReducers, applyMiddleware } from 'redux';
 import logger from 'redux-logger';
+import thunk from 'redux-thunk';
 import booksReducer from './books/books';
 
 const reducer = combineReducers({
   books: booksReducer,
   // additional reducers could be added here
 });
 
+// const getBooks = async () => {
+//   const pull = await fetch(url);
+//   await pull.json();
+// };
+
+// const sendBooks = async (books) => {
+//   await fetch(url, {
+//     method: 'POST',
+//     headers: {
+//       'Content-type': 'application/json',
+//     },
+//     body: JSON.stringify(books),
+//   });
+// }
```

Comment:
Please remove commented code from here as it can ultimately make the whole project unclean.